project/backend/doc-server/src/main/resources/files/dd398660-ca95-4b58-97c8-5c7c5d2a8a31.py

```
@@ -0,0 +1,128 @@
+    query = request_data['query']
+    query_embedding = embed_query(query)
+
+    relevant_chunks, distances = get_context_from_faiss(context_id, query_embedding)
+    if not relevant_chunks:
+        return jsonify({'error': f'No context found for context ID {context_id}'}), 404
+
+    response = query_gemini(relevant_chunks, query)
+    return jsonify(response), 200
+
+if __name__ == '_main_':
+    app.run(debug=True)
```

Ruff (0.8.2)

Use of `debug=True` in Flask app detected

(S201)